src/pthread-main.js

```diff
@@ -26,31 +29,32 @@ var Module = {};
 
 function threadPrint() {
   var text = Array.prototype.slice.call(arguments).join(' ');
-  postMessage({cmd: 'print', text: text, threadId: selfThreadId});
+  console.log(text);
 }
 function threadPrintErr() {
   var text = Array.prototype.slice.call(arguments).join(' ');
-  postMessage({cmd: 'printErr', text: text, threadId: selfThreadId});
+  console.error(text);
 }
 function threadAlert() {
   var text = Array.prototype.slice.call(arguments).join(' ');
   postMessage({cmd: 'alert', text: text, threadId: selfThreadId});
 }
 Module['print'] = threadPrint;
 Module['printErr'] = threadPrintErr;
-
-// Work around https://bugzilla.mozilla.org/show_bug.cgi?id=1049091
-console = {
-  log: threadPrint,
-  error: threadPrintErr
-};
-
```

Comment:
The above change removes an old workaround that is no longer needed. Unrelated, but was helpful to do this to ease debugging (removing this causes console logs appear earlier since they don't need to be queued via postMessage)